packages/relay/src/lib/eth.ts

```diff
@@ -172,7 +172,9 @@ export class EthImpl implements Eth {
 
     try {
       const latestBlockNumber = await this.translateBlockTag(EthImpl.blockLatest, requestId);
-      const newestBlockNumber = await this.translateBlockTag(newestBlock, requestId);
+      const newestBlockNumber = (newestBlock == EthImpl.blockLatest || newestBlock == EthImpl.blockPending)
```

Comment:
Please add a test that ensure the previous case does not occur.
That is a test based on the logic that
- if `newestBlock == EthImpl.blockLatest` then `newestBlockNumber` = `latestBlockNumber`
- if `newestBlock == EthImpl.blockPending` then `newestBlockNumber` = `this.translateBlockTag(newestBlock, requestId)`

Reply:
> if newestBlock == EthImpl.blockPending then newestBlockNumber =

Should not be also = `latestBlockNumber` or do you mean that this case should not happen?

Reply:
Good question, since a call to translateBlockTag with a block in a pending state should also return the latest block.

Reply:
Good question since the call to translateBlockTag with a newestBlock in a pending state will return the "most recent block number" by calling  the getLatestBlock on the mirrorNodeClient.  The tests seem return the same result when passed "latest" or "pending".